frontend/src/pages/clerical-review/details/ClericalMain.css

```
@@ -1,10 +1,7 @@
 .body-clerical {
-    position: fixed;
     top: 0;
     left: 0;
-    background-color: #A2C0BD;
-    height: 100vh;
-    width: 100%;
+    width: 80vw;
```

```suggestion
    width: 80vw;
```

Vaut mieux utiliser des valeurs relatives (%) ici